pygmt/tests/test_contour.py

```diff
@@ -83,7 +83,7 @@ def test_contour_vec(region):
     y = y.flatten()
```

Comment:
@michaelgrund Although it makes no differences, could you also please update `projection="4i"` in the `test_contour_fail_no_data` test?

Reply:
Also update `pen=""` to `pen=True`, as the latter is more Pythonic and readable.

Reply:
Done, thanks for mentioning 😉 @seisman.